app/bundles/PageBundle/Entity/Hit.php

```diff
@@ -258,6 +261,11 @@ public static function loadMetadata (ORM\ClassMetadata $metadata)
             ->build();
 
         $builder->addNullableField('query', 'array');
+
+        $builder->createOneToOne('deviceStat', 'Mautic\LeadBundle\Entity\StatDevice')
```

Comment:
The unique index created from this relationship needs to be added to the migration file. 

`ALTER TABLE mautic_page_hits UNIQUE INDEX UNIQ_7C64CE802373B993 (devicestat_id);` 

Use `$this->generatePropertyName('page_hits', 'uniq', [devicestat_id])` for the name